src/plugins/intel_cpu/src/config.cpp

```diff
@@ -371,6 +371,7 @@ void Config::readProperties(const ov::AnyMap& prop, const ModelType modelType) {
                                ov::hint::kv_cache_precision.name(),
                                ". Supported values: u8, bf16, f16, f32");
             }
+        } else if (key == ov::internal::caching_with_mmap.name()) {
```

Comment:
since it's read-only property, I suppose we don't need to handle it here

Reply:
We should to avoid exception in default block.